src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/azcopy/__init__.py

```diff
@@ -0,0 +1,4 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
This will increase the  installer payload a lot, I suggest we download per demand (which the async is invoked). Doable?
In the same way like 'az aks install-cli'

Reply:
Updated.